src/main/java/springboot/service/impl/BookServiceImpl.java

```diff
@@ -12,14 +16,26 @@
 public class BookServiceImpl implements BookService {
 
     private final BookRepository bookRepository;
+    private final BookMapper bookMapper;
 
     @Override
-    public Book save(Book book) {
-        return bookRepository.save(book);
+    public BookDto createBook(CreateBookRequestDto requestDto) {
+        return bookRepository.createBook(requestDto);
```

Comment:
use mapper here to map dto to book

Reply:
here should I use the save method that returns Book or createBook ?